h2o-py/h2o/estimators/gam.py

```diff
@@ -860,48 +861,48 @@ def custom_metric_func(self, custom_metric_func):
 
 
     @property
-    def k(self):
+    def num_knots(self):
         """
         Number of knots for gam predictors
 
         Type: ``List[int]``.
         """
-        return self._parms.get("k")
+        return self._parms.get("num_knots")
 
-    @k.setter
-    def k(self, k):
-        assert_is_type(k, None, [int])
-        self._parms["k"] = k
+    @num_knots.setter
+    def num_knots(self, num_knots):
+        assert_is_type(num_knots, None, [int])
+        self._parms["num_knots"] = num_knots
 
 
     @property
-    def knots_keys(self):
+    def knot_ids(self):
         """
         String arrays storing frame keys of knots.  One for each gam column specified in gam_x
```

Comment:
>>String arrays storing frame keys of knots.  One for each gam column specified in gam_x
Change to
String arrays storing frame keys of knots.  One for each GAM column specified in gam_columns